src/testing/reduced_order.cpp

```diff
@@ -0,0 +1,37 @@
+#include <iostream>
+
+#include "reduced_order.h"
+#include "parameters/all_parameters.h"
+#include "pod/pod.h"
+
+
+namespace PHiLiP {
+namespace Tests {
+
+template <int dim, int nstate>
+ReducedOrder<dim, nstate>::ReducedOrder(const PHiLiP::Parameters::AllParameters *const parameters_input)
+        : TestsBase::TestsBase(parameters_input)
+{}
+
+template <int dim, int nstate>
+int ReducedOrder<dim, nstate>::run_test() const
+{
+    int num_basis = 8;
+
+    ProperOrthogonalDecomposition::POD podtest(num_basis);
+    podtest.get_full_pod_basis();
+
+    dealii::FullMatrix<double> podbasis(podtest.svd_u->n_rows(), num_basis);
+    podbasis = podtest.get_reduced_pod_basis();
+
+    std::ofstream out_file("pod_basis_matrix.txt");
+    podbasis.print_formatted(out_file, 4);
+    out_file.close();
+
+    return 0;
```

Comment:
Until there is a better checking condition, I would remove the test from the CMakeLists for the sake of this PR.

In addition, this test depends on another test to generate its input data files. This means that if this test is run without the other it will fail. If the previous test was run with old data files, this may succeed. Aka, this test passing or failing will depend on previous runs of another test. I think that ideally, either you have 1 test that generates data AND does a POD. Or you have one test that generates data, for which you check for some correctness, and another test that does POD based on previously generated files. Those files can be hosted in some cloud service that would be downloaded through CMake or something.

Reply:
That's a good point. I think it may be easier to have 1 test that generates data and does a POD. This is something I could do after I actually implement POD. 

For now, I will remove the test from the CMakeLists as you suggest.